chia/full_node/mempool.py

```diff
@@ -417,7 +417,8 @@ def create_bundle_from_mempool_items(
             if not item_inclusion_filter(name):
                 continue
             try:
-                cost = uint64(0 if item.npc_result.conds is None else item.npc_result.conds.cost)
+                assert item.npc_result.conds is not None
+                cost = item.npc_result.conds.cost
```

Comment:
What guarantees that `.conds` is never `None` here?

Reply:
I don't believe we add mempool items if the spendbundle fail to validate (which is the case where conditions would be `None`)